src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams_asset_filter_scenarios.py

```diff
@@ -0,0 +1,266 @@
+
+        self.kwargs.update({
+            'amsname': amsname,
+            'storageAccount': storage_account_for_show,
+            'location': 'westus2',
+            'container': container
+        })
+
+        self.cmd('az ams account create -n {amsname} -g {rg} --storage-account {storageAccount} -l {location}')
+        self.cmd('az ams account storage add -a {amsname} -g {rg} -n {storageAccount}')
+
+        assetName = self.create_random_name(prefix='asset', length=12)
+        alternateId = self.create_random_name(prefix='aid', length=12)
+        description = self.create_random_name(prefix='desc', length=12)
+
+        self.kwargs.update({
+            'assetName': assetName,
+            'alternateId': alternateId,
+            'description': description
+        })
+
+        self.cmd('az ams asset create -a {amsname} -n {assetName} -g {rg} --description {description} --alternate-id {alternateId} --storage-account {storageAccount} --container {container}')
+
+        filterName = self.create_random_name(prefix='filter', length=12)
+
+        self.kwargs.update({
+            'filterName': filterName,
+            'bitrate': 420,
+            'endTimestamp': 100000000,
+            'liveBackoffDuration': 60,
+            'presentationWindowDuration': 1200000000,
+            'startTimestamp': 40000000,
+            'timescale': 10000000,
+            'tracks': '@' + self._get_test_data_file('filterTracks.json'),
+        })
+
+        self.cmd('az ams asset-filter create -a {amsname} --asset-name {assetName} -g {rg} -n {filterName} --bitrate {bitrate} --end-timestamp {endTimestamp} --live-backoff-duration {liveBackoffDuration} --presentation-window-duration {presentationWindowDuration} --start-timestamp {startTimestamp} --timescale {timescale} --tracks "{tracks}"')
+
+        self.cmd('az ams asset-filter show -a {amsname} --asset-name {assetName} -g {rg} -n {filterName}', checks=[
+            self.check('firstQuality.bitrate', '{bitrate}'),
+            self.check('name', '{filterName}'),
+            self.check('presentationTimeRange.endTimestamp', '{endTimestamp}'),
+            self.check('presentationTimeRange.liveBackoffDuration', '{liveBackoffDuration}'),
+            self.check('presentationTimeRange.presentationWindowDuration', '{presentationWindowDuration}'),
+            self.check('presentationTimeRange.startTimestamp', '{startTimestamp}'),
+            self.check('presentationTimeRange.timescale', '{timescale}'),
+            self.check('tracks[0].trackSelections[0].operation', 'Equal'),
+            self.check('tracks[0].trackSelections[0].property', 'FourCC'),
+            self.check('tracks[0].trackSelections[0].value', 'AVC1'),
+            self.check('tracks[1].trackSelections[0].operation', 'NotEqual'),
+            self.check('tracks[1].trackSelections[0].property', 'Unknown'),
+            self.check('tracks[1].trackSelections[0].value', 'AVC1'),
```

Comment:
If you look at the file, this assert shouldn't pass, it's EC-3, not AVC1

Reply:
depending on whichever branch is merged first, either i or you will have to make changes